fhir-bench-orchestrator/src/test_framework/mod.rs

```diff
@@ -212,14 +211,150 @@ impl std::fmt::Display for ServerOperationName {
 /// Details the performance of a single server operation, across all iterations (including any failures).
 #[derive(Deserialize, SerdeValue, Clone, Serialize)]
 pub struct ServerOperationMetrics {
-    pub throughput_rpm: Decimal,
-    pub latency_mean: Decimal,
-    pub latency_p50: Decimal,
-    pub latency_p90: Decimal,
-    pub latency_p99: Decimal,
-    pub latency_p999: Decimal,
-    pub latency_p100: Decimal,
-    pub latency_hdr_histogram: String, // FIXME type
+    pub throughput_per_second: f64,
+    pub latency_millis_mean: f64,
+    pub latency_millis_p50: u64,
+    pub latency_millis_p90: u64,
+    pub latency_millis_p99: u64,
+    pub latency_millis_p999: u64,
+    pub latency_millis_p100: u64,
+    #[serde(with = "serde_histogram")]
+    pub latency_histogram: Histogram<u64>,
+    pub latency_histogram_hgrm_gzip: String,
```

Comment:
This field seems a bit silly, to have in the struct. I wonder if serde supports deriving extra fields for an object?

Nevertheless, not worth fixing now.